code/lib/cli/src/project_types.ts

```diff
@@ -63,8 +63,8 @@ export const SUPPORTED_RENDERERS: SupportedRenderers[] = [
 export enum ProjectType {
   UNDETECTED = 'UNDETECTED',
   UNSUPPORTED = 'UNSUPPORTED',
-  REACT_SCRIPTS = 'REACT_SCRIPTS',
   REACT = 'REACT',
+  REACT_SCRIPTS = 'REACT_SCRIPTS',
```

Comment:
What's the story here?

Reply:
We have a big error count regarding "Storybook does not support react-scripts < 5.0.0" and this error can also be thrown when you choose react-scripts from the list of projects to install, but you don't have react-scripts installed.

I suspect what is influencing this count so much is that people just press enter when they get a list of options, and react-scripts is the first in the list. By changing the order, people will get `react` instead, free of the possibility of crashing with the react-scripts issue.